pom.xml

```diff
@@ -35,6 +36,17 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <!-- remove these log4j once spring boot upgrade is available after 23/12/2021 -->
+            <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-api</artifactId>
+                <version>2.16.0</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-to-slf4j</artifactId>
+                <version>2.16.0</version>
+            </dependency>
```

Comment:
I don't think you need these anymore? Only if log4j-core then we would need to add a dependancy. For lfp-appeals-api, no log4j dependancies were added

Reply:
I need to override dependency coming from `spring-boot-starter-test`
```
[INFO] +- org.springframework.boot:spring-boot-starter-test:jar:2.1.15.RELEASE:test
[INFO] |  +- org.springframework.boot:spring-boot-starter:jar:2.1.15.RELEASE:compile
[INFO] |  |  +- org.springframework.boot:spring-boot:jar:2.1.15.RELEASE:compile
[INFO] |  |  +- org.springframework.boot:spring-boot-autoconfigure:jar:2.1.15.RELEASE:compile
[INFO] |  |  +- org.springframework.boot:spring-boot-starter-logging:jar:2.1.15.RELEASE:compile
[INFO] |  |  |  +- org.apache.logging.log4j:log4j-to-slf4j:jar:2.11.2:compile
[INFO] |  |  |  |  \- org.apache.logging.log4j:log4j-api:jar:2.11.2:compile
[INFO] |  |  |  \- org.slf4j:jul-to-slf4j:jar:1.7.30:compile
```

```
PD15974:bankrupt-officer-search-api mzaouali$ mvn dependency:list | grep log4j
[INFO]    org.apache.logging.log4j:log4j-api:jar:2.11.2:compile
[INFO]    org.apache.logging.log4j:log4j-to-slf4j:jar:2.11.2:compile
````

Reply:
You are right @mreis-ch I will remove them as from Steve, the `log4j-to-slf4j` and `log4j-api` jars are both fine at any version as long as you don't have a version of  `log4j-core`  on the path.